src/events.ts

```diff
@@ -302,7 +302,7 @@ const updateEventResult = (
 export const eventsHandler =
   (context, extrinsicChain: Chain, expectedEvents: Event[], resolve, reject) =>
   async ({ events = [], status }) => {
```

Comment:
I believe we can just use this events, instead of querying them via api.
Trying it